### PR TITLE
Services

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/index.mdx [hunk 1]
```diff
@@ -338,6 +338,25 @@ dates and times.
...
+<Box title="Services" to="services">
+Introduction to services.
+</Box>
+
+<Box title="Toit" to="services/toit">
+A tutorial on how to communicate between different Toit containers.
+</Box>
+
+<Box title="C" to="services/c">
+A tutorial on how to communicate between Toit and C containers.
```
Comment (kasperl) [inline docs/tutorials/index.mdx:355]: It is pretty crude for now, but it probably makes sense to document what we have.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/tutorials/services/index.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+it.
+
+## Serialization
+The communication between clients and providers is done by serializing
+the arguments and return values of the methods. The serialization
+format is based on
+[tison](https://libs.toit.io/encoding/tison/library-summary)
+and thus accepts `null`, integers, booleans, floats, strings, byte arrays,
+lists and maps.
+
```
[the remark this answers is not in the document]
Comment (floitsch) [reply docs/tutorials/services/index.mdx]: Absolutely. I wouldn't know anything about it.

**Comment by floitsch** [comment]
Abandoning.

**Pull request closed without merging**